packages/dashboard/src/components/pages/trade-type/cfd-test.tsx

```diff
@@ -0,0 +1,109 @@
+import * as React from 'react';
+import AppToTrade from '../../template/app-to-trade';
+import Instruments from '../../template/instruments';
+import Indices from '../../template/indices';
+import YouMightLike from '../../template/you-might-like';
+import Banner from '../../template/banner';
+
+const card_data = [
```

Comment:
A general suggestion: maybe we can replace `sub_title` with `subtitle`, this looks a bit better in my opinion

Reply:
I don't know why I used underline ! :D